.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 // Place your settings in this file to overwrite default and user settings.
 {
+    "editor.formatOnSave": false, // Something in the default VSCode is formatting differently than tsfmt wants to
```

Comment:
what does formatOnSave do?

Reply:
When I would save a `.ts` file, VSCode would format the file. I disabled _every_ extension I had and it was still formatting. The only problem was that it was formatting contrary to `tsfmt`, which is enforced in CI. It was just easier to make sure VSCode didn't try to be clever than format the file back and forth.